package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
It seems that TS 2.1 is still not supported for AOT: https://github.com/angular/angular/pull/13294